### ISSUE TITLE
Buying multiple tickets or purchasing as an org does not ask for ticket holder info

### ISSUE THREAD

**Issue by RCheesley** [opencollective/opencollective#3860]
**Describe the bug**
This was a major pain point for us when ticketing for MautiCon and prevented us from automating any part of our ticketing to event workflow.

When either an organisation purchases a ticket, or anybody purchased multiple tickets (individual or organisation) we did not know who the actual ticket holder should be. 

This made it a very laborious process for us to add them to our event platform (ideally we would do this via the API) because:

1. We first had to determine if it was an **individual** or an **organisation** (only available on one of the csv exports)
2. Then we had to determine if **one** or **multiple** tickets were purchased (not easy to spot on flexible pricing) - only available on the other CSV export to point 1)
3. Then we had to ask Support for the **main organisation contact** where an organisation on Open Collective was used to purrchase, as this was not on any export or available via the API
4. Then we had to manually reach out to organisations (having got the email address from Support) and/or the multi ticket purchasers to confirm who the ticket should be assigned to.
5. Often the person/s who were the org contact were not the ticket holder and sometimes were no longer at the company or had no knowledge of the ticket purchase

**Expected behavior**
Here is a suggestion for addressing this issue:
![screenshot-opencollective com-2021 01 18-18_59_38](https://user-images.githubusercontent.com/2930593/104954652-8b43c780-59c0-11eb-91e1-6097fcddb50c.png)

Expand a fieldset per ticket purchased asking for the ticket holder first name / last name / email address. Associate this against the ticket ID as ticket holders.

**Comment by Betree** [comment]
As a quick hack, something we can do is to add an optional textarea for these tickets with a description like `If more than one person is participating, please write the list of names below`.

**Comment by RCheesley** [comment]
Yes that would work - may not play too nicely with API calls (we need to push ticket purchases over to our event platform) but would allow for a manual cross check at least.

**Comment by RCheesley** [comment]
Maybe if we were to specify a format (eg a comma separated list) we could work some magic though, thinking about it.

**Comment by iamronen** [comment]
@RCheesley is this issue still pertinent to you? 
has the nature of the problem changed since this issue was created?
have any solutions surfaced directly/indirectly to address this?

**Comment by RCheesley** [comment]
@iamronen I don't know - it was such a significant problem that we moved entirely away from using Open Collective ticketing as a result :(

As far as I know you can't ask any custom questions when someone purchases a ticket, and you don't have the option to ask an org purchaser the name of the ticket holder, so it must still be an issue - but please do correct me if I am wrong there!
